submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/SubmissionApiTest.kt

```diff
@@ -103,5 +115,28 @@ internal class SubmissionApiTest(tempFolder: TemporaryFolder) : BaseIntegrationT
             assertThat(savedSubmission).isNotNull
             assertThat(savedSubmission).isEqualTo(submission)
         }
+
+        @Test
+        fun `get projects`() {
+            val accNo = "SampleProject1"
+            val title = "Sample Project"
+            val accessTag = tagsDataRepository.save(AccessTag(name = accNo))
+            val submission = Submission(accNo = accNo)
```

Comment:
For tests, we usually use the submission DSL like this:
https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/SubmissionApiTest.kt#L61